update.sh

```diff
@@ -1095,6 +1095,21 @@ setup_auto_update() {
     fi
 }
 
+check_free_space() {
+    dockerRootDir="$(docker info --format='{{ print .DockerRootDir }}')"
+    freeSpace="$(df --block-size=1M "${dockerRootDir}" | tail -1 | awk '{print $4}')"
+
+    if [[ "${freeSpace}" -lt 800 ]]; then
+        echo "There is not enough disk space available to safely update UCRM. At least 800 MB is required for update."
+        echo "You have ${freeSpace} MB of available disk space in ${dockerRootDir}"
+        echo -e "\n----------------\n"
+        echo "We recommend running \"docker system prune\" once in a while to clean unused containers, images, etc."
+        echo "You can determine how much space can be cleaned up by running \"docker system df\""
```

Comment:
Perhaps we want to run that here, too.

Reply:
We don't actually, it's just recommendation and does not have to be related to the available space at all, just something to check. This message is actually shown at the end of each update.
We only automatically prune old UCRM images.